src/api/app/views/webui2/webui/request/show.html.haml

```diff
@@ -9,7 +9,15 @@
     %h5
       Diff
   .card-body
-    -# Diff content and build results
+    - @actions.each_with_index do |action, action_index|
+      - action[:sourcediff].each_with_index do |sourcediff, _sourcediff_index|
```

Comment:
why are you using `each_with_index` if you are not using the index? :rofl:

Reply:
:man_facepalming: Changed `each_with_index` to `each`, and removed the index.